doc/content/components/network-server.md

```diff
@@ -7,3 +7,51 @@ weight: 3
 The Network Server handles the LoRaWAN network layer, including MAC commands, regional parameters and adaptive data rate (ADR).
 
 <!--more-->
+
+## Device Management
+
+Network Servers expose `Get`, `Set` and `Delete` gRPC methods for device management. REST analogues for these methods exist as well. Typical clients of these RPC methods are [Console]({{< ref "/components/console.md" >}}) and [CLI]({{< ref "/components/cli.md" >}}).
+
+In case the device has an active session and it is in Class B or C, a downlink task may added to internal downlink task queue following the device update.
+
+## Application Downlink Queue Management
+
+Network Servers expose `DownlinkQueueReplace`, `DownlinkQueuePush` and `DownlinkQueueList` gRPC methods for application downlink management. Typical clients of these RPC methods are [Application Servers]({{< ref "/components/application-server.md" >}}).
```

Comment:
Probably same as above; no need for rpc names, but a link to the API would be helpful, and the fact that NS exposes downlink queue replace, push and list operations to AS.